### PR TITLE
Update credentials playbook

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [x] @devops team member has commented with :+1:
  - [X] are you adding any new default values that need to be overridden when this goes live?  
    - [X] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [X] Add an entry to the CHANGELOG.

Relevant DEVOPS Ticket: https://openedx.atlassian.net/browse/DEVOPS-5289
Relevant Credentials PR: https://github.com/edx/credentials/pull/147
@edx/ecommerce @clintonb 

Note: I haven't touched this repo before, so I'm not 100% sure about these changes, but I made this change based on what course discovery does with edx_catalog_extensions

### CODE REVIEW

**Review by clintonb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -208,3 +208,6 @@
 - Role: edxapp
   - Added `XQUEUE_RABBITMQ_VHOST` to allow configuring the xqueue RabbitMQ host.
   - Added `XQUEUE_RABBITMQ_PORT` and `XQUEUE_RABBITMQ_TLS` to allow configuring the RabbitMQ port, and enabling TLS respectively.
+
+- Role: credentials
+  - Added `EXTRA_APPS` to enable edX specific data in credentials
```
Comment (clintonb) [inline CHANGELOG.md]: ...to enable the inclusion of additional Django apps in the Credentials Service.

**Review by rlucioni — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -208,3 +208,6 @@
 - Role: edxapp
   - Added `XQUEUE_RABBITMQ_VHOST` to allow configuring the xqueue RabbitMQ host.
   - Added `XQUEUE_RABBITMQ_PORT` and `XQUEUE_RABBITMQ_TLS` to allow configuring the RabbitMQ port, and enabling TLS respectively.
+
```
Comment (feanil) [inline CHANGELOG.md]: New changelog entries should go at the top.

**Review by feanil — CHANGES_REQUESTED** [review]
Other than changelog, this looks good.

**Comment by MatthewPiatetsky** [comment]
@feanil 
did I fix it?  
Assuming I also need to wait for the devops ticket to be complete for this to be merged?

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@MatthewPiatetsky Since this has a default value(empty list) you don't need to wait on the secure config before merging this.  Though you will need to wait on that ticket before deploying this code to our production system.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+- Role: credentials
+  - Added `EXTRA_APPS` to enable the inclusion of additional Django apps in the Credentials Service.
```
Comment (feanil) [inline CHANGELOG.md]: This should be `CREDENTIALS_EXTRA_APPS` which is the top level var that was added.

**Pull request merged**